### PR TITLE
[RM Ch04]: Changing NFVI to Cloud Infrastructure

### LINKED ISSUE #1168: [RM] generic requirements of CNF in RM3.3 and RM4.1
@tomkivlin we have some observation in RM stream . It looks the requirements are too much VNF focused and not scoped adequately for CNF .

E.g 4.1 Capabilities and Performance Measurements we can find VIM performance but not  K8S platform capabilities part 

3.3 NFVI Management Software is another example we expect something like container management as well 
Similarly for K8S at minimum performance requirements , CRD's and CRI-O etc should be included

I am sure it is included somewhere can you ignore my negligence if so and help us understand how you are tacking it .

### PR DESCRIPTION
**Pull request by CsatariGergely**
- NFVI -> Cloud Infrastructure
- VNF -> workload
- Figures updated

Partially fixes #1168

### CODE REVIEW

**Comment by kedmison** [comment]
@pgoyal01 @karinesevilla you will need to see this as the move to change nfvi to cloud infrastructure is having consequences on the metric/attribute naming scheme.

**Comment by CsatariGergely** [comment]
Should be merged only after #1372

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -175,129 +174,129 @@ This section covers a list of implicit NFVI capabilities and measurements that d
...
-| e.vim.cap.003 | Virtual Networking resources allocation | Yes/No | Capability to allocate virtual networking resources  to VNFC |
-| e.vim.cap.004 | Multi-tenant isolation                  | Yes/No | Capability to isolate resources between tenants              |
-| e.vim.cap.005 | Images management                       | Yes/No | Capability to manage VNFC software images                    |
+| Ref       | Cloud Infrastructure management Capability | Unit   | Definition/Notes                                                 |
+|-----------|--------------------------------------------|--------|------------------------------------------------------------------|
+| e.man.001 | Virtual Compute allocation                 | Yes/No | Capability to allocate virtual compute resources to a workload   |
+| e.man.002 | Virtual Storage allocation                 | Yes/No | Capability to allocate virtual storage resourcesto a workload    |
+| e.man.003 | Virtual Networking resources allocation    | Yes/No | Capability to allocate virtual networking resourcesto a workload |
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter04.md]: Two typos resourcsto didn't make it into the review....

File: doc/ref_model/chapters/chapter04.md [hunk 2]
```diff
@@ -175,129 +174,129 @@ This section covers a list of implicit NFVI capabilities and measurements that d
...
+**Table 4-7** below shows possible performance optimisation capabilities that can be provided by Cloud Infrastructure. These include capabilities exposed to workloads as well as internal capabilities to Cloud Infrastructure. These capabilities will be determined by the Cloud Infrastructure Profile used by the Cloud Infrastructure.
 
 <a name="Table4-7"></a>
 
-| Ref            | NFVI capability   | Unit   | Definition/Notes                      |
-|----------------|-------------------|--------|---------------------------------------|
-| i.nfvi.cap.018 | Huge page support | Yes/No | Indicates if NFVI supports huge pages |
+| Ref       | Cloud Infrastructure capability | Unit   | Definition/Notes                      |
+|-----------|---------------------------------|--------|---------------------------------------|
+| i.cap.018 | Huge page support               | Yes/No | Indicates if NFVI supports huge pages |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter04.md]: NFVI In the Definitions

File: doc/ref_model/chapters/chapter04.md [hunk 3]
```diff
@@ -399,18 +398,18 @@ The following table defines persistent storage extensions that can be provided t
...
+### 4.2.4 Cloud Infrastructure Profiles
 
 <a name="4.2.4.1"></a>
 #### 4.2.4.1 Basic Profile
-This NFVI Profile is intended to be used for both IT workloads as well as NFV workloads. It has limited IO capabilities (up to 10Gbps Network interface).
+This Cloud Infrastructure Profile is intended to be used for both IT workloads as well as NFV workloads. It has limited IO capabilities (up to 10Gbps Network interface).
 
 <a name="4.2.4.2"></a>
 #### 4.2.4.2 Network Intensive Profile
 This NFVI Profile is intended to be used for those applications that has high network throughput requirements (up to 50Gbps).
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter04.md]: "This NFVI profile is ..."

File: doc/ref_model/chapters/chapter04.md [hunk 4]
```diff
@@ -30,143 +30,142 @@ This section describes and uniquely identifies the Capabilities provided directl
...
-| i.nfvi.cap.014 | CPU cores consumed by NFVI overhead in a compute node | % (of total available) | Indicates the percentage of cores consumed by NFVI components (including host OS) in a compute node  |
-| i.nfvi.cap.015 | Memory consumed by NFVI overhead in a compute node    | % (of total available) | Indicates the percentage of memory consumed by NFVI components (including host OS) in a compute node |
+| Ref       | Cloud Infrastructure Capability                       | Unit                   | Definition/Notes                                                   |
+|-----------|-------------------------------------------------------|------------------------|--------------------------------------------------------|
+| i.cap.014 | CPU cores consumed by NFVI overhead in a compute node | % (of total available) | Indicates the percentage of cores consumed by NFVI components (including host OS) in a compute node  |
+| i.cap.015 | Memory consumed by NFVI overhead in a compute node    | % (of total available) | Indicates the percentage of memory consumed by NFVI components (including host OS) in a compute node  |
```
[the remark this answers is not in the document]
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter04.md]: Fixed

File: doc/ref_model/chapters/chapter04.md (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter04.md]: Fixed

File: doc/ref_model/chapters/chapter04.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter04.md]: Fixed

File: doc/ref_model/chapters/chapter04.md (hunk 3 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter04.md]: Fixed

**Pull request merged**